holoviews/tests/conftest.py

```diff
@@ -80,3 +82,17 @@ def plotly_backend():
     hv.Store.current_backend = 'plotly'
     yield
     hv.Store.current_backend = prev_backend
+
+@pytest.fixture
+def unimport(monkeypatch: pytest.MonkeyPatch) -> Callable[[str], None]:
+    """
+    Return a function for unimporting modules and preventing reimport.
+    """
+
+    def unimport_module(modname: str) -> None:
+        # Remove if already imported
+        monkeypatch.delitem(sys.modules, modname, raising=False)
+        # Prevent import:
+        monkeypatch.setattr(sys, "path", [])
```

Comment:
That's kind of crazy, but if it works it works :) Could you add a comment in the docstring saying that it prevents importing anything really, not just `modname`.

Reply:
Will do.